src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -150,8 +171,45 @@ Domain<3> BinaryCompactObject::create_domain() const noexcept {
   std::move(maps_cube_B.begin(), maps_cube_B.end(), std::back_inserter(maps));
   std::move(maps_frustums.begin(), maps_frustums.end(),
             std::back_inserter(maps));
-  std::move(maps_outer_shell.begin(), maps_outer_shell.end(),
-            std::back_inserter(maps));
+
+  if (outer_spherical_shell_layers_ > 1) {
+    // wedge_coordinate_maps() can only include more than one layer
+    // if the sphericity is constant. Therefore, we surround the outermost
+    // cubed-sphere (made as a single layer with sphericity varying from 0.0
+    // on its inner boundary to 1.0 on its outer boundary) with
+    // (outer_spherical_shell_layers_) - 1 layers each with constant
+    // sphericity == 1
+    const double inner_radius_first_outer_shell =
+        sqrt(3.0) * 0.5 * length_outer_cube_;
+    const double outer_radius_first_outer_shell =
+        use_logarithmic_map_outer_spherical_shell_
```

Comment:
Were you able to add a test that caught that the log/lin were interchanged here? @gsb76 any suggestions on how to test that?

Reply:
Ooh this one might be tricky, if one doesn't know which layer a block is coming from. I would think one can grab a random point on the +zeta face of a block and get its radius in the grid frame. One can then do this for three blocks (at least) in different layers and then check whether the radii are growing linearly or geometrically. Then this is checked to be consistent with the setting for `use_logarithmic_map`